### ISSUE TITLE
Revamping for looper v2

### ISSUE THREAD

**Issue by nsheff** [pepkit/hello_looper#19]
In writing the looper v2 docs, I'm updating some of these. I've created a few new examples, which were based on what we had before, but with a few minor tweaks. Here are some suggestions I think we should do:

- [x] remove 'toggle' from hello world examples. It's confusing to me and not clear why it's there in even the most basic examples.
- [x] make the minimal example == to the csv example (I think of the csv example as the minimal example)
- [x] make clear in the README, and in the example names, what the different examples show. For example, this is not helpful: "/intermediate - An intermediate example pipeline and project with a couple extra options." -- What would be more helpful would be: "/XandY_example - An example pipeline and project that demonstrates how to use options X and Y."
- [x] rename advanced example? I'm not sure what it demonstrates based on the name, I guess
- [x] I think it's worth updating the remaining examples (pephub and pipestat) to use this new framework I'm building with the countries, because it is a little more engaging than the random frog example, while still being small and fast.

I renamed `csv` to `looper_csv_example`, so that the name matches what I used in the tutorial.
Then, I named my second example as `pep_derived_attrs`, because that's what I named in the tutorial.

Basically, this repository will just be the place where the "completed" tutorial files will live, in case someone doesn't want to actually do the "paste this here" or whatever steps to the tutorial, and just see the final thing.

**Comment by donaldcampbelljr** [comment]
The advanced example comes from the original pytest example for looper. I pulled it into hello looper since our tests now rely more heavily on the examples found in hello_looper (this was to attempt to ensure they were kept in-sync). It features multiple pipelines as well as using sample_modifiers, derive attributes. It could potentially be removed in the future, but it will probably break some tests and require them to be rewritten.

**Comment by donaldcampbelljr** [comment]
Similarly to the above comment, toggle is included in examples to help with pytesting basic features. We could definitely remove toggle from most examples and keep it in one for this purpose.

**Comment by nsheff** [comment]
I see. That make sense, but I think we should at least separate the examples that are intended for users to learn on, and examples that are intended for testing.

So, maybe make some pytest-oriented examples if needed, and mark them as such, but then restrict the other ones to only what's important for the tutorial.

**Comment by donaldcampbelljr** [comment]
Yeah, I'm in agreement with that approach.

**Comment by donaldcampbelljr** [comment]
Alright, I refactored these per the above suggestions.
